magento/magento2
# 26214: Bundle Product / Create / Price validation for dynamic price

### Description (*)
- Fixed Bundle Product / Create / Price validation for dynamic price

### Fixed Issues (if relevant)

1. magento/magento2#26214: Bundle Product / Create / Price validation for dynamic price

### Manual testing scenarios (*)
1. Enter "New Product / Bundle" form
2. Turn off "Dynamic price"
3. Enter the price and remove it. You'll see a validation error.
4. Turn on "Dynamic price". You'll see remove a validation error.

video fixed https://youtu.be/zyPJw6udqO0


### Contribution checklist (*)
